README.md

```diff
@@ -264,16 +292,46 @@ Get all Services in staging.east:
     4.rails.staging.east.skydns.local. 3600 IN A    10.0.1.125
     6.rails.staging.east.skydns.local. 3600 IN AAAA 2003::8:1
 
+If you ask for a service who's Host value is an IP address you would (in theory) get
+back a SRV record such as:
+
+    % dig @localhost 4.rails.staging.east.skydns.local SRV
+
+    ;; ANSWER SECTION:
+    4.rails.staging.east.skydns.local 3600 IN SRV 10 100 8080 10.0.1.125
+
+Where the target of the SRV is an IP address. This is not how SRV records work.
+SkyDNS will in this case synthesize a domain name and add the actual IP
+address to the additional section of the response:
+
+    % dig @localhost 4.rails.staging.east.skydns.local SRV
+
+    ;; ANSWER SECTION:
+    4.rails.staging.east.skydns.local 3600 IN SRV 10 100 4.rails.staging.east.skydns.local.
+
+    ;; ADDITIONAL SECTION:
+    4.rails.staging.east.skydns.local. 3600 IN A    10.0.1.125
+
+Which conveys the same information and is legal in the DNS. To have some control on how
+the target names look you can register a service with `TargetStrip` set to a none-zero
+value. This tells SkyDNS to strip `TargetStrip` labels from the left-side of the generated
+Target domain name. Support `TargetStrip` is 2 for this service, to above response would be:
+
```

Comment:
Not clear what TargetStrip does - suggest:
Setting TargetStrip to "2" strips 2 labels from the generated target name.  (if that is what it does)